husky_dual_ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
+
+  <arg name="debug" default="false" />
+  <arg unless="$(arg debug)" name="launch_prefix" value="" />
+  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
+
+  <arg name="config" default="false" />
+  <arg unless="$(arg config)" name="command_args" value="" />
+  <arg     if="$(arg config)" name="command_args" value="-d $(find husky_dual_ur5_moveit_config)/launch/moveit.rviz" />
+  
+  <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
+	args="$(arg command_args)" output="screen">
+    <rosparam command="load" file="$(find husky_dual_ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="l_ur5_arm_"/>
```

Comment:
I changed the way the universal_robot kinematics plugin acts, so this is unnecessary @TheDash TODO

@ibaranov-cp https://github.com/ros-industrial/universal_robot/pull/248/files